### DISCUSSION TITLE
Create a MSFT teams app to have a selected budget as a Teams channel tab

### DISCUSSION THREAD

**Discussion by BerndGewehr** [openkfw/TruBudget#872] [Ideas]
This way you could combine the non-structured conversation with the structured trusted documentation in one place which would allow a better and more agile style of collaboration inside the project teams. Have the update transactions or due date notifications alert to the channel feed for easy recognizing what’s happening and what’s due.

**Comment by BerndGewehr** [comment]
Maybe better use azure ad SSO for easy federated login from within teams. Look for Adal library to start easy.

**Comment by jzakotnik** [comment]
Thanks @BerndGewehr for the idea.. I remember that BNDES did an SAP adapter in their trubudget repo, maybe we should think about an Teams Adapter talking to the API (without touching core TruBudget)
